awswrangler/s3/_write_parquet.py

```diff
@@ -542,6 +537,19 @@ def to_parquet(  # pylint: disable=too-many-arguments,too-many-locals,too-many-b
     }
 
     """
+    glue_catalog_parameters = cast(
+        GlueCatalogParameters,
+        glue_catalog_parameters if glue_catalog_parameters else {},
+    )
+
+    table_type = glue_catalog_parameters.get("table_type")
+    transaction_id = glue_catalog_parameters.get("transaction_id")
+    description = glue_catalog_parameters.get("description")
+    parameters = glue_catalog_parameters.get("parameters")
+    columns_comments = glue_catalog_parameters.get("columns_comments")
```

Comment:
I would vote `catalog_id` be pulled back into root parameters. Reasoning being it is more similar to something like `table` or `database` than the params here.